pkg/cmd/util_content_test.go

```diff
@@ -97,3 +98,12 @@ func TestIsBinary(t *testing.T) {
 	assert.True(t, isBinary("application/zip"))
 	assert.False(t, isBinary("text/plain"))
 }
+
+func TestContentHttp(t *testing.T) {
+	u, err := url.Parse("https://raw.githubusercontent.com/apache/camel-k/main/go.mod")
```

Comment:
to test this, I had to call a remote service that may sometimes fail
is this acceptable ?

Reply:
I'm a bit worried about [rate limiting](https://github.community/t/raw-githubusercontent-com-rate-limit/142444). I wonder whether the `httptest` package could provide a quick way to cover this, maybe with: https://pkg.go.dev/net/http/httptest#NewServer?

Reply:
yep, good point
let me rework to use the server

Reply:
done

Reply:
faster than light 🚀!